### PR TITLE
feat(db): seed db

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
At present, we have some type mismatches and also, we have implemented methods to get data from db but no actual data + connection yet. 

## Solution
Seed the db with data

### CODE REVIEW

File: apps/studio/prisma/generated/generatedTypes.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@ export type Blob = {
 }
 export type Footer = {
   id: string
-  name: string
```
Comment (kishore03109) [inline apps/studio/prisma/generated/generatedTypes.ts:16]: why are we dropping this ah?

File: apps/studio/src/schemas/site.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { z } from 'zod'
+
+export const getConfigSchema = z.object({
+  id: z.string(),
```
Comment (kishore03109) [inline apps/studio/src/schemas/site.ts:4]:
```suggestion
  id: z.string().min(1),
```

File: apps/studio/src/server/modules/site/site.types.ts [hunk 3]
```diff
@@ -1,11 +1,11 @@
 export interface Sitemap {
   parentTitle: string
   childrenTitles: string[]
-  sibilngTitles: string[]
+  siblingTitles: string[]
 }
-
+type SiteTheme = 'isomer-classic' | 'isomer-next'
```
Comment (kishore03109) [inline apps/studio/src/server/modules/site/site.types.ts:6]: nit: obj this so that in code can just call `SiteTheme.classic`

**Review by kishore03109 — APPROVED** [review]

File: apps/studio/prisma/generated/generatedTypes.ts (hunk 1 above)
Comment (seaerchin) [reply apps/studio/prisma/generated/generatedTypes.ts:16]: this is an extra prop. i referenced the footer props in config: 

```ts

{
  LinkComponent?: any
  isGovernment?: boolean
  siteName: string
  agencyName: string
  lastUpdated: string
}
```

File: apps/studio/src/schemas/site.ts (hunk 2 above)
Comment (seaerchin) [reply apps/studio/src/schemas/site.ts:4]: updated

File: apps/studio/src/server/modules/site/site.types.ts (hunk 3 above)
Comment (seaerchin) [reply apps/studio/src/server/modules/site/site.types.ts:6]: sure

**Pull request merged**